arch/xtensa/core/xtensa-asm2.c

```diff
@@ -10,12 +10,18 @@
 #include <zephyr/kernel_structs.h>
 #include <kernel_internal.h>
 #include <kswap.h>
-#include <_soc_inthandlers.h>
 #include <zephyr/toolchain.h>
 #include <zephyr/logging/log.h>
 #include <offsets.h>
 #include <zsr.h>
 
+#ifdef CONFIG_XTENSA_GEN_HANDLERS
```

Comment:
This `CONFIG_XTENSA_GEN_HANDLERS` is nowhere defined so, by default, `xtensa_handlers.h` is used.